app/assets/stylesheets/components/pill.scss

```
@@ -88,7 +88,7 @@
 
     display: block;
     text-align: left;
-    padding: 10px $gutter-half;
+    padding: 10px govuk-spacing(3);
```

`10px` is equivalent to `$gutter-one-third`
```suggestion
    padding: govuk-spacing(2) govuk-spacing(3);
```